### ISSUE TITLE
When running atria v3.2.2 with switch --stats, encounter error not seen in v3.2.1

### ISSUE THREAD

**Issue by kalavattam** [cihga39871/Atria#13]
Hi, just want to report what seems to be a bug.

When running atria v3.2.2 with switch `--stats`, I encounter the following error:
```sh
"nested task error: UndefVarError: is_concensused not defined"
```

When I remove `--stats` from my call to atria v3.2.2, the program runs fine. When I call atria v3.2.1 with `--stats`, the program runs fine.

I noticed a variable `is_concensused` used in [wrapper.jl#L385](https://github.com/cihga39871/Atria/blob/master/src/Trimmer/wrapper.jl#L385) and a variable `is_consensused` in [consensus.jl](https://github.com/cihga39871/Atria/blob/master/src/FqRecords/consensus.jl). Note the minor difference in spelling between the two variables: con<u><i><b>c</b></i></u> in `is_concensused` and  con<u><i><b>s</b></i></u> `is_consensused`. I wonder if this is related to the error?

A minimal example to reproduce the error:
```sh
#!/bin/bash

threads=8
fq_1="SRR7939018_1.fastq.gz"
fq_2="SRR7939018_2.fastq.gz"
d_trim="."

#  Obtain relatively small fastqs for testing
curl "ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR793/008/SRR7939018/${fq_1}" > "${fq_1}"
curl "ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR793/008/SRR7939018/${fq_2}" > "${fq_2}"

#  Run v3.2.2 with --stats: Encounter error
alias atria="\${HOME}/path/to/Atria/app-3.2.2/bin/atria"  # Change path as appropriate
atria \
    -t "${threads}" \
    -r "${fq_1}" \
    -R "${fq_2}" \
    -o "${d_trim}" \
    --no-length-filtration \
    --stats

#  Run v3.2.2 without --stats: Runs fine
atria \
    -t "${threads}" \
    -r "${fq_1}" \
    -R "${fq_2}" \
    -o "${d_trim}" \
    --no-length-filtration

#  Run v3.2.1 with --stats: Runs fine
alias atria="\${HOME}/path/to/Atria/app-3.2.1/bin/atria"  # Change path as appropriate
atria \
    -t "${threads}" \
    -r "${fq_1}" \
    -R "${fq_2}" \
    -o "${d_trim}" \
    --no-length-filtration \
    --stats

#  Run v3.2.1 without --stats: Runs fine
atria \
    -t "${threads}" \
    -r "${fq_1}" \
    -R "${fq_2}" \
    -o "${d_trim}" \
    --no-length-filtration
```

**Comment by cihga39871** [comment]
Thank you @kalavattam for reporting this! After investigating, I found the cause of the undef error: `is_concensused` is defined after a `@goto` statement. I fixed it in v3.2.2-1.

**Comment by cihga39871** [comment]
Also, the typo was fixed.
